README.md

```diff
@@ -1,3 +1,10 @@
+> Warning
+> 
+> <u>This crate is no longer updated as it has been moved to <a href="https://crates.io/crates/dioxus-sdk">dioxus-sdk</a>.</u>
```

Comment:
```suggestion
> [!NOTE]
> The create has been renamed from `dioxus-std` to `dioxus-sdk` as of April 3rd, 2024.
```

Reply:
we can change the date later on

Reply:
![image](https://github.com/DioxusLabs/dioxus-std/assets/38158676/3e1cd2eb-2819-4f16-bd9f-f738984aa9b7)
 this is how it looks